### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by tjbenton**

### CODE REVIEW

File: app/cli.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
+import commander from 'commander';
+import Fakeit from './index.js';
+import updateNotifier from 'update-notifier';
+import pkg from './../package.json';
+import path from 'path';
+import { extend, flattenDeep, pick, find } from 'lodash';
+import chalk from 'chalk';
+
+export default async function() {
```
Comment (bentonam) [inline app/cli.js]: missing docs comments

File: app/logger.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+      }
+
+      const stamp = this.stamp(type);
+
+      // print the current time.
+      if (stamp) {
+        process.stdout.write(stamp);
+      }
+
+      console.log(arg);
```
Comment (bentonam) [inline app/logger.js:89]: Remove this console.log

File: app/models.js [hunk 3]
```diff
@@ -1,308 +1,505 @@
...
+  return inputs;
+}
+
+/// @name parseModelFunctions
+/// @description
+/// searches the model for any of the pre / post run and build functions and generates them
+/// @arg {object} model - The model to update
+/// @arg {string, object} babel_config [{}] - The configuration to use for babel
+export function parseModelFunctions(model, babel_config = {}) {
+  // console.log('models.parseModelFunctions');
```
Comment (bentonam) [inline app/models.js]: remove this line

File: app/models.js [hunk 4]
```diff
@@ -1,308 +1,505 @@
...
-    let property = objectPath.get(models[model], type_path);
+}
+
+/// @name parseModelReferences
+/// @description
+/// searches the model for any '$ref' values that are pointing to definitions,
+/// sub_models, etc. and copies the reference to the schema
+/// @arg {object} model - The model to update
+export function parseModelReferences(model) {
+  // console.log('models.parseModelReferences');
```
Comment (bentonam) [inline app/models.js]: remove this line

File: app/models.js [hunk 5]
```diff
@@ -1,308 +1,505 @@
...
+  }
+}
+
+/// @name parseModelTypes
+/// @description
+/// Searches the model for any properties or items and makes
+/// sure the default types exist
+/// @arg {object} model - The model to update
+export function parseModelTypes(model) {
+  // console.log('models.parseModel_properties');
```
Comment (bentonam) [inline app/models.js]: remove this line

File: app/models.js [hunk 6]
```diff
@@ -1,308 +1,505 @@
...
+  for (let type_path of utils.objectSearch(model, /.*properties\.[^.]+(\.items)?$/)) {
+    // console.log(type_path);
```
Comment (bentonam) [inline app/models.js]: remove this line

**Review by bentonam — CHANGES_REQUESTED** [review]
Remove a few console.log's and commented code

File: app/logger.js (hunk 2 above)
Comment (tjbenton) [reply app/logger.js:89]: required to log stuff to the console

File: app/models.js (hunk 3 above)
Comment (tjbenton) [reply app/models.js]: done

File: app/models.js (hunk 4 above)
Comment (tjbenton) [reply app/models.js]: done

File: app/models.js (hunk 5 above)
Comment (tjbenton) [reply app/models.js]: done

File: app/models.js (hunk 6 above)
Comment (tjbenton) [reply app/models.js]: done

File: app/cli.js (hunk 1 above)
Comment (tjbenton) [reply app/cli.js]: done

**Review by bentonam — APPROVED** [review]

**Pull request merged**